libs/libarchfpga/src/physical_types.h

```diff
@@ -626,14 +610,52 @@ struct t_physical_tile_type {
 
     int index = -1; /* index of type descriptor in array (allows for index referencing) */
 
-    int logical_block_index = -1; /* index of the corresponding logical block type */
+    std::vector<std::string> equivalent_sites_names;
+    std::vector<t_logical_block_type_ptr> equivalent_sites;
```

Comment:
It could actually be eliminated. `equivalent_sites_names` was necessary to link physical tiles to logical blocks, but I have found a better way to do that which could let me avoid using the extra `equivalent_sites_names`